proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -0,0 +1,540 @@
+
+  * Set [`internal`] to the result of
+    [`color(channel1 channel2 channel3 / alpha)`]
+
+* Otherwise, set [`internal`] to the result of
+  [`color(channel1 channel2 channel3 )`]
+
+[`color(channel1 channel2 channel3 / alpha)`]: ./color-4-new-spaces.md#color-1
+[`color(channel1 channel2 channel3 )`]: ./color-4-new-spaces.md#color-1
+
+[construction]: ../spec/js-api/value/color.d.ts.md#constructor
+
+#### RGB Channel Constructor
+
+Create a new SassColor in a color space with RGB channels -- `rgb`, `srgb`,
+`srgb-linear`, `display-p3`, `a98-rgb`, and `prophoto-rgb`. If `space` is set,
+it will create a new SassColor in that space, and it will default to the legacy
+`rgb` space.
+
+```ts
+constructor(
+  options: {
+    red: ChannelValue;
+    green: ChannelValue;
+    blue: ChannelValue;
+    alpha?: number;
+  },
+  space?: ColorSpaceRGB
+);
+```
+
+#### HSL Channel Constructor
+
+Create a new SassColor in the `hsl` color space. `space` is optional to not
+break the legacy constructor, but allowed for constructor consistency.
+
+```ts
+constructor(
+  options: {
+    hue: ChannelValue;
+    saturation: ChannelValue;
+    lightness: ChannelValue;
+    alpha?: number;
+  },
+  space?: ColorSpaceHSL
+);
+```
+
+#### HWB Channel Constructor
+
+Create a new SassColor in the `hwb` color space. `space` is optional to not
+break the legacy constructor, but allowed for constructor consistency.
+
+```ts
+constructor(
+  options: {
+    hue: ChannelValue;
+    whiteness: ChannelValue;
+    blackness: ChannelValue;
+    alpha?: number;
+  },
+  space?: ColorSpaceHWB
+);
+```
+
+#### XYZ Channel Constructor
+
+Create a new SassColor in a color space with XYZ channels -- `xyz`, `xyz-d50`,
+and `xyz-d65`. `space` is required as there is no legacy space to default to.
+
+```ts
+constructor(
+  options: {
+    x: ChannelValue;
+    y: ChannelValue;
+    z: ChannelValue;
+    alpha?: number;
+  },
+  space: ColorSpaceXYZ
+);
+```
+
+#### LCH Channel Constructor
+
+Create a new SassColor in a color space with LCH channels -- `lch` and `oklch`.
+`space` is required as there is no legacy space to default to.
+
+```ts
+constructor(
+  options: {
+    lightness: ChannelValue;
+    chroma: ChannelValue;
+    hue: ChannelValue;
+    alpha?: number;
+  },
+  space: ColorSpaceLCH
+);
+```
+
+#### LAB Channel Constructor
+
+Create a new SassColor in a color space with LAB channels -- `lab` and `oklab`.
+`space` is required as there is no legacy space to default to.
+
+```ts
+constructor(
+  options: {
+    lightness: ChannelValue;
+    a: ChannelValue;
+    b: ChannelValue;
+    alpha?: number;
+  },
+  space: ColorSpaceLAB
+);
+```
+
+```ts
+}
+```
+
+### Deprecations
+
+A number of SassColor getters only make sense for legacy colors, and so are
+being deprecated for `channel`.
+
+* `red`
+* `green`
+* `blue`
+* `hue`
+* `saturation`
+* `lightness`
+* `whiteness`
+* `blackness`
+* `alpha`
+
+In addition, `change` is deprecated in favor of `changeChannels`.
```

Comment:
Does this need to be renamed? Isn't `change()` broadly forward-compatible with `changeChannels()`?

Reply:
Addressed in c54902d